auk-notebook.ipynb

```diff
@@ -645,9 +712,19 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "plt.rcParams['figure.figsize'] = [10, 4] # Set the figure size for the graph.\n",
-    "\n",
-    "NETWORK_EXCLUDE = [\"google.com\"]\n",
+    "NETWORK_PLOT_FIGURE_SIZE = [10, 4] # Set the figure for the graph\n",
```

Comment:
Same thing, one more cell above. Just one more markdown cell above the variables.

Along the lines of:

```
Change the variables in the following cell to manipulate the network analysis that you'll be carrying out. Some of these relate to the size of the output, the labels on the graph, and also any domains you might want to exclude from your analysis.
```